app/services/displayable_pii_formatter.rb

```diff
@@ -43,11 +43,7 @@ def all_emails
 
   def verified_at
```

Comment:
Possible follow-up item: If we're okay to not show it some of the time, should we show it none of the time? In other words, should we get rid of it altogether if it's not PII?

Reply:
the spirit of this is, we're showing users data that we are sharing with a partner, so I vote to keep it some of the time

I am guessing the absence of the attribute is what confused users, leading to this ticket?

Reply:
> the spirit of this is, we're showing users data that we are sharing with a partner, so I vote to keep it some of the time

I'd agree, and I'd think we also have to inform them we're sending the data in all cases anyway? We're telling an SP that the person is not verified, which is a piece of information about them.

Reply:
There's additional context for the background in the ticket, re: consent considerations.